doc/user-changes.md

```diff
@@ -4,6 +4,20 @@ This document is a development diary summarizing changes in `alr` that notably
 affect the user experience. It is intended as a one-stop point for users to
 stay on top of `alr` new features.
 
+### Show release-specific dependency sets in solutions
```

Comment:
BTW @mosteo, this is now live on the website: https://alire.ada.dev/docs/#user-facing-changes-log

Reply:
Great! I had indeed forgotten about updating the website... :S